spec/features/sign_up_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+
+feature  'i want to be able to sign up' do
+
+
+
+ scenario "user clicks on the submit button" do
```

Comment:
Watch your spacing here, scenario should start on line 5 and be indented 2 spaces from feature is (rather than 2) and your 'end's should also line up so the first end should match the indentation of scenario and the second end should line up with feature.